src/components/Organisms/AlertManager/index.jsx

```diff
@@ -0,0 +1,23 @@
+import React, { useEffect } from "react";
+import { useAlertHandlerContext } from "../../../contexts/alert_handler";
+import Alert from "../../Atoms/Alert";
+
+const AlertManager = () => {
+  const AlertHandlerContext = useAlertHandlerContext();
+
+  useEffect(() => {
+    if (AlertHandlerContext.visible === true) {
```

Comment:
```suggestion
    if (AlertHandlerContext.visible) {
```
値がbooleanの場合はこのように書くと直感的になるかなと思います。